src/main/java/UI.java

```diff
@@ -0,0 +1,23 @@
+public class UI {
+    public static String logo = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+
+    public static String divider = "-----------------------------------";
```

Comment:
Since the Strings referenced by both "logo" and "divider" variables will never change, perhaps you can consider to declare them as constants, by capitalizing both variable names and adding the "final" keyword to them.

Reply:
Good point; will change this! Thank you :)